### PR TITLE
Nikon D3S: reenable camera support after check, fixes #11247, #11232

### LINKED ISSUE #11247: local copy icon status and culling / full view
**Describe the bug/issue**

The local status icon gets displayed when in culling or full view on the top-left corner of the image area.

**To Reproduce**
1. Go to 'lighttable'
2. Select an image
3. Click on 'copy locally'
4. Click on x for culling - see wrong local copy status icon displayed
5. Click on w for full view - see wrong local copy status icon displayed

**Expected behavior**

I believe the local status icon should be skipped on those views. But to be discussed.

**Screenshots**

Full-view:

Capture d’écran de 2022-03-01 18-56-16

Culling:

Capture d’écran de 2022-03-01 18-56-32

### PR DESCRIPTION
**Pull request by schenlap**
12bit was already enabled. Fix sensor crop for all modes.

### CODE REVIEW

File: src/external/rawspeed/data/cameras.xml [hunk 1]
```diff
@@ -1965,27 +1965,27 @@
...
 		<ID make="Nikon" model="D3S">Nikon D3S</ID>
 		<CFA width="2" height="2">
 			<Color x="0" y="0">RED</Color>
 			<Color x="1" y="0">GREEN</Color>
 			<Color x="0" y="1">GREEN</Color>
 			<Color x="1" y="1">BLUE</Color>
 		</CFA>
-		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="0" white="16384"/>
+		<Crop x="2" y="0" width="0" height="0"/>
```
Comment (LebedevRI) [inline src/external/rawspeed/data/cameras.xml]: Oh, nice catch :/

**Comment by LebedevRI** [comment]
@schenlap thank you!

**Comment by LebedevRI** [comment]
@schenlap 
E8400 sample - https://redmine.darktable.org/issues/8479 - "12bit-uncompressed" is the only mode acc. to PDF usermanual

**Comment by schenlap** [comment]
@LebedevRI 
Server mention in  https://redmine.darktable.org/issues/8479 is not reachable.

**Comment by LebedevRI** [comment]
Ugh, i thought i checked.
@schenlap i guess we are done with the white levels then (until the owners of last 4 cameras surface), thank you very much!

**Pull request merged**